### ISSUE TITLE
DOI

### ISSUE THREAD

**Issue by t-pollington** [jbracher/hhh4addon#9]
Hi Johannes,

I was going to cite your package at this repo (I'm also citing your methodological papers elsewhere) and was wondering if you had a DOI for this repository?

Kind regards, @t-pollington 
```
@misc{Bracher2020b,
author = {Bracher, Johannes},
publisher = {GitHub},
title = {\texttt{hhh4addon}: development package arising out of the \texttt{surveillance} package},
url = {github.com/jbracher/hhh4addon},
year = {2020}
}
```

**Comment by jbracher** [comment]
Hi Tim, thanks for asking. There is not currently, but I can generate one. I'll let you know here.

**Comment by jbracher** [comment]
I finally got around to this: https://zenodo.org/record/4696126#.YHlckhKxVH4. As you've made a couple of PRs your name even appears in the entry ;)

**Comment by t-pollington** [comment]
Great, thanks for that @jbracher , I've updated my manuscript with it. It may seem a bit pedantic but I recently learnt how this [article](https://academic.oup.com/jid/article/198/5/687/865839) published in 2008 lost its online appendix because the journal had a 10 year archive limit! So now on, irrespective of the journal's current policy I'm going to make sure I use these additional archival sites. In fact in my citation of your package it's just you! Firstly the analysis is still based on the run of the code last year when you were the sole author (I always include the version numbers of packages I use for better reproducibility). Plus even if I had used the new code I'd have probably just taken off my name if all it was was a typo PR!, since it's pretty open to abuse eh? Though I think where someone has spotted a bug and suggested a change via a PR, which I did for `spatstat` a while ago, then that makes sense.
